tck/src/main/java/jakarta/json/bind/tck/cdi/customizedmapping/adapters/AdaptersCustomizationCDITest.java

```diff
@@ -48,18 +45,26 @@
 /*
  * @class.setup_props: webServerHost; webServerPort; ts_home;
  */
-@RunWith(Arquillian.class)
 public class AdaptersCustomizationCDITest {
-    
-    @Deployment
-    public static WebArchive createTestArchive() {
-        return ShrinkWrap.create(WebArchive.class)
-                .addPackages(true, MethodHandles.lookup().lookupClass().getPackage().getName(),
-                        "jakarta.json.bind.tck.customizedmapping.adapters.model")
-                .addAsWebResource(EmptyAsset.INSTANCE, ArchivePaths.create("beans.xml"));
+
+    private static SeContainer container;
+
+    @BeforeAll
+    private static void startContainer() {
+        try {
+            //Verify that CDI container is already running
+            CDI.current();
```

Comment:
Note that the Platform TCK [vehicle.properties](https://github.com/eclipse-ee4j/jakartaee-tck/blob/master/install/jsonb/other/vehicle.properties#L53) indicated that servlet was needed for Standalone jsonb 2.0 TCK which looks wrong since the JSONB 2.0 spec doesn't mention that requirement.  So this seems fine to remove testing with servlet.  

Any disagreement?

Reply:
I agree that servlet does not make sense as there aren't any servlet tests to run.

Reply:
AFAIK to be JSONB certified you don't need servlet run, to be EE certified for the JSONB part you need servlet, ejb, ejb embed, jsp and jsf runs too. Know some spec limited that list to standalone/servlet as a shortcut but think it is worth testing in a EE container *for the platform* otherwise it is like not having any test - but fine to move this TCK part in the platform TCK and not in the jsonb tck bundle.

Reply:
> AFAIK to be JSONB certified you don't need servlet run, to be EE certified for the JSONB part you need servlet, ejb, ejb embed, jsp and jsf runs too. Know some spec limited that list to standalone/servlet as a shortcut but think it is worth testing in a EE container _for the platform_ otherwise it is like not having any test - but fine to move this TCK part in the platform TCK and not in the jsonb tck bundle.

Technically, we can leave jsonb tests in the Platform TCK for EE 10 but in the long term, it would be good to have a single source for JSONB tests.  Perhaps once we have finished refactoring the Jakarta EE TCK tests (e.g. switch to multiple Maven modules) we can then achieve that.

Reply:
These two TCK bundles (in `jakartatck` and in this repository) are different. This bundle contains tests for all newly added features and has better tests coverage. We are not going to maintain two bundles because we cannot have two sources of truth. We should remove all JSONB tests from `jakartatck` repository or leave only tests related to CDI. @scottmarlow, do you want me to file an issue for this?

Reply:
> These two TCK bundles (in `jakartatck` and in this repository) are different. This bundle contains tests for all newly added features and has better tests coverage. We are not going to maintain two bundles because we cannot have two sources of truth. We should remove all JSONB tests from `jakartatck` repository or leave only tests related to CDI. 

^ Makes sense but I think the Platform TCK still needs to validate any Platform requirements that the new JSONB TCK doesn't validate.

@scottmarlow, do you want me to file an issue for this?

Yes, please, I think that the removal of the JSONB TCK tests from the Platform TCK needs to avoid removing tests that validate the Platform requirements.  Perhaps this could be to just leave the CDI tests in place (option 1), or we could leave a few (small number) of other JSONB tests in the Platform TCK as well to provide more coverage (option 2).